src/Cake.Common.Tests/Unit/Build/ContinuaCI/Data/ContinuaCIBuildInfoTests.cs

```diff
@@ -0,0 +1,308 @@
+﻿using Cake.Common.Tests.Fixtures.Build;
```

Comment:
Mixed tabs and spaces in this file. Change to spaces and re-indent file (CTRL+K+D in Visual Studio).